simulation/state.go

```diff
@@ -166,10 +135,93 @@ func AppStateFn(
 		// we should replace the EvmDenom with BondDenom
 		evmState.Params.EvmDenom = stakingState.Params.BondDenom
 
+		observers := make([]string, 0)
+		// Get all the operator addresses of the validators.
+		// The observer set can be a subset of the validator set
```

Comment:
```suggestion
		// get all the operator addresses of the validators.
		// the observer set can be a subset of the validator set
```
Can we use lowercase everywhere in comments